### PR TITLE
Data is now being saved in mongodb from frontend

### PR DESCRIPTION
**Pull request by prabal01**
I and Deepak implemented this backend task...Now data is being saved in the database please check the code and let us know if there are any changes required

### CODE REVIEW

File: client/src/Components/create_post/add_post/add_post.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+function AddPost() {
+	const [file, setFile]=useState("filename")
+	const [author, setAuthor]=useState("")
+	const [caption, setCaption]=useState("")
+	const [location , setLocation]=useState("")
+	
+	const locationChangeHandler=(e)=>{
+		setLocation(e.target.value)
+		console.log("location: ",location)
```
Comment (jasce) [inline client/src/Components/create_post/add_post/add_post.js]: remove console.log statements

File: client/src/Components/landing_page/landing_page.css [hunk 2]
```diff
@@ -10,8 +8,9 @@ a {
 .landingPage {
 	display: flex;
 	padding-top: 5vh;
-	padding-bottom: 5vh;
+	/* padding-bottom: 5vh; */
```
Comment (jasce) [inline client/src/Components/landing_page/landing_page.css:11]: remove commented code

File: client/src/Components/navigation_bar/header.css [hunk 3]
```diff
@@ -20,12 +23,19 @@
...
-	margin-left: 5px;
+	padding-left: 15px;
 	margin-bottom: 0px;
 }
 .camera {
 	width: 36px;
 	height: 36px;
 	margin: 26px;
-	padding-right: 200px;
+	/* padding-right: 200px; */
```
Comment (jasce) [inline client/src/Components/navigation_bar/header.css:33]: remove

File: client/src/Components/post_component/post_list/post_list.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+					Name="siva"
+					place="Bengaluru"
+					image="https://i.ytimg.com/vi/5sTg4Oeh2rM/maxresdefault.jpg"
+					date="10 Jan 2021"
+					likes="64"
+					heading="Kick start your career"
+				/>
+
+				<PostCard
+					Name="Sameer"
```
Comment (jasce) [inline client/src/Components/post_component/post_list/post_list.js]: prop name should not be capitalised

File: routes/index.js [hunk 5]
```diff
@@ -1,13 +1,52 @@
...
+//     console.log("file:" ,req.file)
+    
+    
+// })
+
+router.post("/upload",  upload.single('file'),function(req,res,next) {
+    var data = {
+        author : req.body.author,
+        location : req.body.location,
+        filePath: req.file.path,
```
Comment (jasce) [inline routes/index.js]: have you checked if you getting the correct image path ??

**Review by jasce — CHANGES_REQUESTED** [review]

File: routes/index.js (hunk 5 above)
Comment (prabal01) [reply routes/index.js]: yes, It's getting the correct path sir

File: client/src/Components/create_post/add_post/add_post.js (hunk 1 above)
Comment (prabal01) [reply client/src/Components/create_post/add_post/add_post.js]: its done sir

File: client/src/Components/landing_page/landing_page.css (hunk 2 above)
Comment (prabal01) [reply client/src/Components/landing_page/landing_page.css:11]: it's done, sir

File: client/src/Components/navigation_bar/header.css (hunk 3 above)
Comment (prabal01) [reply client/src/Components/navigation_bar/header.css:33]: it's done, sir

File: client/src/Components/post_component/post_list/post_list.js (hunk 4 above)
Comment (prabal01) [reply client/src/Components/post_component/post_list/post_list.js]: it's done, sir

**Review by jasce — APPROVED** [review]

**Pull request merged**